### PR TITLE
Convert inline script to a Stimulus controller

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

File: app/components/throbber.html.erb [hunk 1]
```diff
@@ -0,0 +1 @@
+<div class="throbber"><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div></div>
```
Comment (mjgiarlo) [inline app/components/throbber.html.erb]: *blinks*

Comment (mjgiarlo) [reply app/components/throbber.html.erb]: @jcoyne 	but srsly, what's going on here? :smiley:

**Comment by mjgiarlo** [comment]
@jcoyne 	can you drop a comment here too about what ActionView::Component is doing for us?

File: app/components/throbber.html.erb (hunk 1 above)
Comment (jcoyne) [reply app/components/throbber.html.erb]: Those divs are the "dots" in the animated throbber

**Comment by jcoyne** [comment]
@mjgiarlo ActionView::Component is server-side component rendering.  It's by github and it's going to be part of rails 6.1.   It's better than view helpers because it's encapsulated and testable. We're already using it in was-registrar-app.

**Pull request merged**